Assets/Scripts/Models/Area/World.cs

```diff
@@ -240,10 +241,11 @@ public Furniture PlaceFurniture(Furniture furniture, Tile t, bool doRoomFloodFil
 
     public void JobComplete_FurnitureBuilding(Job theJob)
     {
-        // Let our workspot tile know it is no longer reserved for us
-        UnreserveTileAsWorkSpot((Furniture)theJob.buildablePrototype, theJob.tile);
+        Furniture furn = (Furniture)theJob.buildablePrototype;
 
-        PlaceFurniture(theJob.JobObjectType, theJob.tile);
+        // Let our workspot tile know it is no longer reserved for us
+        UnreserveTileAsWorkSpot(furn, theJob.tile);
+        PlaceFurniture(theJob.JobObjectType, theJob.tile, true, furn.Rotation);
```

Comment:
should we just use furn at this point? is already a clone... and we might not even need to clone it again. What do you think?